src/components/Input.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function Input({ value, onChange }) {
+  return <input type="number" value={value} onChange={onChange} />;
+}
```

Comment:
Do not hardcode the type, you can pass the remaining props to the component instead, it will support all input HTML attributes like so:

By the way, technically you don't have to restructure `onChange`, and `value`, you could spread everything directly but for the learning purpose let's keep it, it will be easier for you to remember which props to pass :)
By this I mean that this would also work: (but for now let's keep on destructuring the props you intend to pass

```js
function Input(props) {
  return <input {...props} />;
}
```

```suggestion
function Input({ value, onChange, ...props }) {
  return <input value={value} onChange={onChange} {...props} />;
}
```